vagrant/provisioning/roles/confluent-platform-install/templates/confluent-control-center.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Confluent Control Center
+Documentation=http://docs.confluent.io/
+After=network.target confluent-kafka.target
+
+[Service]
+Type=simple
+User=cp-control-center
+Group=confluent
+Environment="LOG_DIR={{root_folder}}/log/control-center" "CONTROL_CENTER_LOG4J_OPTS=-Dlog4j.configuration=file:{{root_folder}}/app/control-center/log4j-rolling.properties"
```

Comment:
Minor - you can have many Environment directives, each with a single environment variable; I find this easier to read.